.github/ISSUE_TEMPLATE/ext-info.yml

```diff
@@ -3,18 +3,71 @@ description: Template to use when sending additional requests for external APIs.
 title: "[Ext]: "
 labels: [external api]
 body:
+  - type: input
+    id: external-api-summary
+    attributes:
+      label: What this API is
+    validations:
+      required: true
+
+  - type: input
+    id: external-api-url
+    attributes:
+      label: API endpoint URL
+    validations:
+      required: true
+
+  - type: textarea
+    id: external-api-list
+    attributes:
+      label: Data you want to get
+      value: |
+        -
+        -
+        -
+    validations:
+      required: true
+
+  - type: input
+    id: external-api-interval
+    attributes:
+      label: Interval you will get data
+      value: times / month
+    validations:
+      required: true
+
+  - type: input
+    id: external-api-type
+    attributes:
+      label: API data type
```

Comment:
I want you to describe the data type (ext) and format.
(I want it to be written in such a way that it is explicitly stated that you are requesting it)